src/main/scala/org/opennetworkinsight/LDAWrapper.scala

```diff
@@ -0,0 +1,172 @@
+package org.opennetworkinsight
```

Comment:
as discussed, in a future checkin, we can break up this file into smaller functions and unit test them